frontend/src/app/models/register.model.ts

```diff
@@ -0,0 +1,7 @@
+export interface IRegister {
```

Comment:
to co poprzednio.
Dlaczego są jako dodatkowe? Przy rejestracji wszystkie pola poza pic są wymagane